chromadb/utils/embedding_functions/jina_embedding_function.py

```diff
@@ -56,10 +51,10 @@ def __call__(self, input: Documents) -> Embeddings:
         if "data" not in resp:
             raise RuntimeError(resp["detail"])
 
-        embeddings = resp["data"]
+        embeddings: List[dict[str, Union[str, List[float]]]] = resp["data"]
```

Comment:
This is similar to the openai one, but here we can cast the expected type as then we are doing `e["index"]`.